shared/actions/fs/index.tsx

```diff
@@ -151,8 +134,8 @@ const getSyncConfigFromRPC = (
     case RPCTypes.FolderSyncMode.partial:
       return Constants.makeTlfSyncPartial({
         enabledPaths: config.paths
-          ? I.List(config.paths.map(str => Types.getPathFromRelative(tlfName, tlfType, str)))
-          : I.List(),
+          ? config.paths.map(str => Types.getPathFromRelative(tlfName, tlfType, str))
+          : [],
```

Comment:
may not matter but just a note. `I.List() === I.List()` and `[] !== []` so either ensure 
1. this doesn't matter (likely)
1. change to accept undefined
1. or make `const emptyPaths = []` and just re-use that

Reply:
Ah, good point. I don't think there's correctness issue here. It'd matter in reducer for performance reasons but we do `isEqual` comparisons there.

I don't think a const emptyPaths would work in a lot of scenarios since we end up doing a `.map` later on and unfortunately `emptyPaths.map(a=>a) !== emptyPaths`